src/ruby/spec/pb/codegen/package_option_spec.rb

```diff
@@ -0,0 +1,53 @@
+# Copyright 2018 gRPC authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+require 'spec_helper'
+require 'open3'
+require 'tmpdir'
+
+describe 'Code Generation Options' do
+  it 'should generate and respect package options' do
+    unless system('which grpc_ruby_plugin') && system('which protoc')
+      skip 'protoc || grpc_ruby_plugin missing, cannot verify package code-gen'
+    end
+
+    root_dir = File.join(File.dirname(__FILE__), '..', '..', '..', '..')
+    pb_dir = File.join(root_dir, 'proto')
+
+    # Regenerate it
+    plugin, = Open3.capture2('which', 'grpc_ruby_plugin')
+    plugin = plugin.strip
+    Dir.mktmpdir(nil, File.dirname(__FILE__)) do |tmp_dir|
+      pid = spawn(
```

Comment:
nit: rather than using `spawn` and `wait`, can we just use `system`, and then also check the return value?